query/src/main/java/org/infinispan/search/mapper/mapping/impl/InfinispanIndexedTypeContextImpl.java

```diff
@@ -83,6 +84,16 @@ public void dirtyFilter(PojoPathFilter dirtyFilter) {
          // Nothing to do
       }
 
+      @Override
+      public void dirtyContainingAssociationFilter(PojoPathFilter filter) {
+         PojoIndexedTypeExtendedMappingCollector.super.dirtyContainingAssociationFilter(filter);
+      }
+
+      @Override
+      public void applyLoadingBinder(Object o, PojoEntityLoadingBindingContext pojoEntityLoadingBindingContext) {
+
+      }
```

Comment:
You can take inspiration from there:

https://github.com/hibernate/hibernate-search/pull/3954/commits/14b61ecf4fb972712528b9fbb889a1a5eec6e92d#diff-1a51fe86a7217bf378d1f17728ccf1eac0b270653736491bcda5a11fb5209af1R61

https://github.com/hibernate/hibernate-search/pull/3954/commits/14b61ecf4fb972712528b9fbb889a1a5eec6e92d#diff-365de9542c6dabaec86c58df4cca84ad57ca120135e952c4ed255633d44e8f10

https://github.com/hibernate/hibernate-search/pull/3954/commits/14b61ecf4fb972712528b9fbb889a1a5eec6e92d#diff-f6f89030fa186954ff2ae3873dbebbf14f212c18ebcdbd67e67a59a24bb32392R143